File: android/sited/src/main/java/org/noear/sited/ISdViewModel.java
package org.noear.sited;

import java.util.Collection;
import java.util.List;

/**
 * Created by yuety on 15/8/2.
 */
public interface ISdViewModel {
    void loadByConfig(SdNode config);
    void loadByJson(SdNode config, String... jsons);
}

File: ios/sited/sited/src/ISdViewModel.swift
//
//  ISdViewModel.swift
//  sited
//
//  Created by 谢月甲 on 15/10/8.
//  Copyright © 2015年 noear. All rights reserved.
//

import Foundation

public protocol ISdViewModel{
    func loadByConfig(config:SdNode);
    func loadByJson(config:SdNode, jsons:[String]!);
}